module/Application/view/error/404.phtml

```diff
@@ -48,7 +48,7 @@ if (isset($this->controller_class)
 
 <?php if (isset($this->display_exceptions) && $this->display_exceptions): ?>
 
-<?php if(isset($this->exception) && $this->exception instanceof Exception): ?>
+<?php if(isset($this->exception) && ($this->exception instanceof Exception || $this->exception instanceof Error)): ?>
```

Comment:
Just a note: we cannot typehint against just `Throwable`, as the skeleton also supports PHP 5.6. As such, testing for either `Exception` or `Error` will satisfy any `Throwable` type from PHP 7.